kylecornelissen/vacation-rentals
# Favorite buttons

What's this PR do?
- Adds favoriting functionality to individual Listings, the ListingsContainer and the Favorites page
- Created a Favorites component to list out favorited listings by the user
- Added routes for Favorites to App.js

How should this be manually tested?
- Go to the listings container and click the star button
- Go to individual listings and click their star buttons
- Go to the favorites link in the NavBar to see the list
- Toggle those if you want

What are the relevant tickets?
#14 

Future Implementation Plans:
- Need to add more links to the Nav Bar to allow user to go back to `/areas`
